artview/components/mapper.py

```diff
@@ -0,0 +1,298 @@
+        self.gridLimitsZmin.setValue(2000)
+        self.gridLimitsZmax = QtWidgets.QDoubleSpinBox()
+        self.gridLimitsZmax.setRange(-41000000, 41000000)
+        self.gridLimitsZmax.setSingleStep(1000)
+        self.gridLimitsZmax.setValue(3000)
+        self.gridLimitsYmin = QtWidgets.QDoubleSpinBox()
+        self.gridLimitsYmin.setRange(-41000000, 41000000)
+        self.gridLimitsYmin.setSingleStep(1000)
+        self.gridLimitsYmin.setValue(-250000)
+        self.gridLimitsYmax = QtWidgets.QDoubleSpinBox()
+        self.gridLimitsYmax.setRange(-41000000, 41000000)
+        self.gridLimitsYmax.setSingleStep(1000)
+        self.gridLimitsYmax.setValue(250000)
+        self.gridLimitsXmin = QtWidgets.QDoubleSpinBox()
+        self.gridLimitsXmin.setRange(-41000000, 41000000)
+        self.gridLimitsXmin.setSingleStep(1000)
+        self.gridLimitsXmin.setValue(-250000)
+        self.gridLimitsXmax = QtWidgets.QDoubleSpinBox()
+        self.gridLimitsXmax.setRange(-41000000, 41000000)
+        self.gridLimitsXmax.setSingleStep(1000)
+        self.gridLimitsXmax.setValue(250000)
+        self.layout.addWidget(QtWidgets.QLabel("grid_limits (m)"), 0, 2)
+        self.layout.addWidget(self.gridLimitsZmin, 1, 2)
+        self.layout.addWidget(self.gridLimitsZmax, 2, 2)
+        self.layout.addWidget(self.gridLimitsYmin, 3, 2)
+        self.layout.addWidget(self.gridLimitsYmax, 4, 2)
+        self.layout.addWidget(self.gridLimitsXmin, 5, 2)
+        self.layout.addWidget(self.gridLimitsXmax, 6, 2)
+
+        self.layout.setRowStretch(8, 1)
+        self.layout.setColumnStretch(1, 1)
+        self.layout.setColumnStretch(2, 1)
+
+    def grid_from_radars(self):
+        '''Mount Options and execute :py:func:`~pyart.correct.grid_from_radars`.
+        The resulting grid is update in Vgrid.
+        '''
+        # test radar
+        if self.Vradar.value is None:
+            common.ShowWarning("Radar is None, can not perform correction")
+            return
+        # mount options
+        self.parameters['radars'] = (self.Vradar.value,)
+        self.parameters['fields'] = []
+        for field in self.field_actions.keys():
+            if self.field_actions[field].isChecked():
+                self.parameters['fields'].append(field)
+
+        self.parameters['grid_shape'] = (self.gridShapeZ.value(),
+                                         self.gridShapeY.value(),
+                                         self.gridShapeX.value())
+        self.parameters['grid_limits'] = (
+            (self.gridLimitsZmin.value(), self.gridLimitsZmax.value()),
+            (self.gridLimitsYmin.value(), self.gridLimitsYmax.value()),
+            (self.gridLimitsXmin.value(), self.gridLimitsXmax.value()))
+
+        self.parameters['grid_origin'] = (self.parameters['grid_origin_lat'],
+                                          self.parameters['grid_origin_lon'])
+
+        # execute
+        print("mapping ..", file=log.debug)
```

Comment:
I wonder if we should start using the [python logging module](https://docs.python.org/2/library/logging.html)